.env.example

```
@@ -26,11 +26,10 @@ SERVER_API_KEY="YOUR_SERVER_API_KEY" # A secret key for your agent server API
 SERVER_PORT="3001"
 
 # --- PostgreSQL Database Configuration (mandatory) ---
-POSTGRES_USER=admin
-POSTGRES_HOST=localhost # dev
-# POSTGRES_HOST=postgres # prod
-POSTGRES_DB=postgres
-POSTGRES_PASSWORD=admin
+POSTGRES_USER="YOUR_USER"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)